### PR TITLE
treewide: remove gebner

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by gebner — APPROVED** [review]

File: pkgs/applications/misc/cura/default.nix [hunk 1]
```diff
@@ -61,6 +61,6 @@ mkDerivation rec {
     homepage = "https://github.com/Ultimaker/Cura";
     license = licenses.lgpl3Plus;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ abbradar gebner ];
```
Comment (SigmaSquadron) [inline pkgs/applications/misc/cura/default.nix:64]: Did you intend to remove more maintainers than just `gebner`? There seem to be many unnecessary maintainer removals.

**Review by SigmaSquadron — CHANGES_REQUESTED** [review]

**Comment by bendlas** [comment]
Closing these removal PRs, since they are not grounded within community consensus. See https://github.com/NixOS/nixpkgs/pull/337478#issuecomment-2310779183

@gebner be sure that we value your involvement and any time you may have to spare, helping with the packages you're maintaining and others as well.

File: pkgs/applications/misc/cura/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/applications/misc/cura/default.nix:64]: No more.

**Comment by linsui** [comment]
> we value your involvement

Interesting that you even don't value gebner's approval.

**Comment by SigmaSquadron** [comment]
> Interesting that you even don't value gebner's approval.

Let's... not escalate. We can just reopen and merge a PR like this. I'm sure it was simply a pre-set response to urandom's PR.

**Comment by SuperSandro2000** [comment]
@gebner do you consider yourself an active maintainer? If not I think it would be cool to reflect that in the meta section of the package.

**Pull request closed without merging**